File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/grades/service/yacontest/YaContestService.java
package org.ilyatyamin.yacontesthelper.grades.service.yacontest;

import org.ilyatyamin.yacontesthelper.dto.yacontest.ContestSubmission;

import java.util.List;

public interface YaContestService {
    List<String> getListOfProblems(String contestId, String yandexAuthKey);
    List<ContestSubmission> getSubmissionList(String contestId, String yandexAuthKey);
}

File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/grades/service/yacontest/YaContestService.kt
package org.ilyatyamin.yacontesthelper.grades.service.yacontest

import org.ilyatyamin.yacontesthelper.grades.dto.ContestSubmission

interface YaContestService {
    fun getListOfProblems(contestId: String?, yandexAuthKey: String?): List<String?>
    fun getSubmissionList(contestId: String?, yandexAuthKey: String?): List<ContestSubmission?>
}
